data/gui/normalStyle.css

```diff
@@ -758,20 +765,26 @@ QScrollBar::left-arrow {
 
 QSlider {
 	background: none;
+	height: 14px;
 }
 
 QSlider::groove:horizontal {
 	border: none;
-	height: 4px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
-	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(0, 0, 0), stop: 1 rgb(54, 54, 54));
+	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
```

Comment:
where is this used? ( cannot find a GUI slider that looks differently now. )

Reply:
This was the point: to avoid any changes (i.e. regressions) at the 100% scale, and at the same time improve the looks when all `px` sizes are multiplied by some factor. This is not used anywhere yet, but I have a WIP branch, [scalable-gui](https://github.com/10110111/stellarium/tree/scalable-gui), where this is important (in this branch [one commit](https://github.com/10110111/stellarium/commit/e6ff0b81e64420dba30e765ead4514f2e0ef72e9) sets a hard-coded 150% factor, which lets me test all this).

Reply:
OK, no further questions. The new color buttons are excellent, thanks!

Reply:
> no further questions

Does this imply you approve the PR?

Reply:
Yes, there is just no "review" button visible after first commenting :-o . Approved.